services-js/commissions-app/src/pages/commissions/apply.tsx

```diff
@@ -5,20 +5,22 @@ import fetchCommissions, {
   Commission,
 } from '../../client/graphql/fetch-commissions';
 
-import { AppLayout, StatusModal } from '@cityofboston/react-fleet';
+import { AppLayout } from '@cityofboston/react-fleet';
 
 import ApplicationForm from '../../client/ApplicationForm';
 import ApplicationSubmitted from '../../client/ApplicationSubmitted';
 import { NextContext } from '@cityofboston/next-client-common';
 import { IncomingMessage } from 'http';
+import { Formik } from 'formik';
+import { ApplyFormValues, applyFormSchema } from '../../lib/validationSchema';
 
 interface Props {
   commissions: Commission[];
   commissionID?: string;
+  testSubmittedPage?: boolean;
 }
 
 interface State {
-  isSubmitting: boolean;
   applicationSubmitted: boolean;
```

Comment:
`isSubmitting` is intended to be used to provide visual feedback to the user while the server is processing their application; I know it's not in the specs, but it might be nice to at least change the cursor to `wait`, and maybe change the submit button text to "submitting...". What do you think?

Reply:
Changed button text to "Submitting…". Added wait cursor as an option to `StatusModal`, since that’s appearing while `isSubmitting` is true.

(BTW the `isSubmitting` that was in the `State` for `ApplyPage` is actually now being tracked by Formik. You can see `isSubmitting` as one of the `FormikProps` that `ApplicationForm` requests.)

Reply:
Ah, missed that in `FormikProps` - thx!